codegen/.snapshots/TestGenerateTemplate-should_add_comma_between_relations

```diff
@@ -1,13 +1,25 @@
-{
-		Name:         "with relations",
-		Columns: []schema.Column{
+// Code generated by codegen; DO NOT EDIT.
 
-		},
+package 
 
-		Relations: []*schema.Table{
-relation1,
-relation2,
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/cloudquery/plugins/source//client"
```

Comment:
Double slash in the import path here?

Reply:
Also, maybe we should gofmt the file after generation so the tests aren't sensitive to whitespace changes